docs/architecture/overview.md

```diff
@@ -1,34 +1,34 @@
 # Architecture Overview
 
-The Carbon Aware SDK was built in 3 layers, with each layer having it's own set of responsibilities. 
-1. User-interface layer
+The Carbon Aware SDK is built with three layers where each layer has its own set of responsibilities.
+1. User interface layer
 2. Business Logic Layer
 3. Data Layer
 
 Each layer only has knowledge of its own responsibilities, and specific channels of communication between adjacent layers. This allows the SDK to switch to different implementations withing the same layer without any necessary code changes.
 
-![Layers architecture diagram](../images/layers-architecture.png)
+![Layers architecture diagram](../images/layers-architecture.drawio.png)
```

Comment:
Love this!  I was able to pull it into draw.io with no issues for editing needs.

Reply:
@JenMadiedo , were you able to edit the file? So it can be updated with the new name for the upper layer?

Reply:
Yes I can edit and will update with push